src/main/java/core/db/DBUpdater.java

```diff
@@ -102,6 +102,8 @@ void updateDB(int DBVersion) {
 						updateDBv300(DBVersion, version);
 						updateDBv301(DBVersion, version);
 					case 302:
+					case 399:
+					case 400:
```

Comment:
I don't understand the logic: `case 400: updateDBv400`   is it not entering into a loop then ? 
To be frank, the all block since case 27 seems a bit strange, we used to have one db version per release and that was simpler to understand.
 Could we not simplify this by merging some of the update function ?   @tychobrailleur what do you think of this ?

Reply:
@akasolace case 400 should be shifted behind the call of updateDBv400. case 399 is necessary to handle the DEV-Versions which are used by us and maybe some brave users. the space in between is left for patch versions of release 3 when version 4 development is in progress.
if you increment only by 1 you give away the chance to deliver patches which needs db updates.